Upstream-Status: Pending

this fix avoids this error with automake 1.12

| automake: warnings are treated as errors
| /srv/home/nitin/builds/build-gcc47/tmp/sysroots/x86_64-linux/usr/share/automake-1.12/am/ltlibrary.am: warning: 'libmpc.la': linking libtool libraries using a non-POSIX
| /srv/home/nitin/builds/build-gcc47/tmp/sysroots/x86_64-linux/usr/share/automake-1.12/am/ltlibrary.am: archiver requires 'AM_PROG_AR' in 'configure.ac'

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/05/02

Index: mpc-0.8.2/configure.ac
===================================================================
--- mpc-0.8.2.orig/configure.ac
+++ mpc-0.8.2/configure.ac
@@ -86,6 +86,10 @@ AC_PROG_EGREP
 AC_PROG_SED
 AC_LANG(C)
 
+# automake 1.12 seems to require this, but automake 1.11 doesn't recognize it
+m4_pattern_allow([AM_PROG_AR])
+AM_PROG_AR
+
 dnl Check for GMP CFLAGS in gmp.h
 if test -z "$user_redefine_cc" ; then
    MPC_GMP_CC_CFLAGS
